packages/app-builder-lib/templates/nsis/assistedInstaller.nsh

```diff
@@ -143,7 +143,11 @@
       !ifdef INSTALL_MODE_PER_ALL_USERS
```

Comment:
I'm usually adverse to having nested if-statements if possible to avoid.

Seems like the logic here could be `INSTALL_MODE_PER_ALL_USERS || INSTALL_MODE_PER_ALL_USERS_DEFAULT`

Is that possible to do in nsis scripting?

Reply:
Unfortunately, it does not seem to be possible with NSIS.